terra/reports.py

```diff
@@ -579,3 +579,170 @@ def merge_data_type(employee_ids, start_date, end_date):
     data["all_type_total"] = complete_total
 
     return data
+
+
+def get_individual_data_employee(employee_ids, start_date=None, end_date=None):
```

Comment:
@aprigge The commit says "changed one typo", but it also adds this large `get_individual_data_employee()` function.  Is this deliberate?

I've confirmed this PR fixes the /unit/1 error with production data, just checking that adding this function here is deliberate and should be included along with the `approval_amount` -> `admin_alloc` change (the presumed typo fix) at line 86.

Reply:
Thanks, @akohler That should be for the employee subtotal. Let me revise this pr so that it is only for TRRA-154.